src/kernels/jupyter/session/jupyterSessionManager.ts

```diff
@@ -354,41 +348,10 @@ export class JupyterSessionManager implements IJupyterSessionManager {
         // Agent is allowed to be set on this object, but ts doesn't like it on RequestInit, so any
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         let requestInit: any = this.requestCreator.getRequestInit();
-        let cookieString;
 
         // If no token is specified prompt for a password
         const isTokenEmpty = connInfo.token === '' || connInfo.token === 'null';
-        if (isTokenEmpty && !connInfo.getAuthHeader) {
```

Comment:
`getAuthHeader` will not be empty for user provided jupyter server either, hence 
there' no need for any of the below code,